flowey/flowey_lib_hvlite/src/init_openvmm_magicpath_openhcl_sysroot.rs

```diff
@@ -76,19 +75,15 @@ impl FlowNode for Node {
                                 OpenvmmSysrootArch::Aarch64 => "aarch64-sysroot",
                                 OpenvmmSysrootArch::X64 => "x86_64-sysroot",
                             });
-                    fs_err::create_dir_all(&extracted_sysroot_path)?;
-
-                    let sh = xshell::Shell::new()?;
-                    xshell::cmd!(
-                        sh,
-                        "tar
-                                -xf {openhcl_sysroot_tar_gz}
-                                -C {extracted_sysroot_path}
-                                --no-same-owner
-                                --no-same-permissions
-                            "
-                    )
-                    .run()?;
+                    fs_err::create_dir_all(extracted_sysroot_path.parent().unwrap())?;
+                    fs_err::remove_dir_all(&extracted_sysroot_path)?;
+
+                    #[cfg(unix)]
+                    let symlink = |orig, link| fs_err::os::unix::fs::symlink(orig, link);
+                    #[cfg(windows)]
+                    let symlink = |orig, link| fs_err::os::windows::fs::symlink_dir(orig, link);
```

Comment:
are you sure we want to take a dep on windows symlink support, given all the weirdness around it?

and in any case - we don't actually use the openhcl sysroot on windows at the moment, this codepath is never going to be hit.

I'd suggest we just bail if we find ourselves on windows, and cross this bridge later

Reply:
I really don't care anymore about Windows environments that don't have developer mode enabled (which among other things, enables symlinks). We can tell people to enable it.

Agreed that we don't currently support Windows. But with flowey+using the sys root, I suspect we are actually fairly close to supporting building purely from Windows. So I don't want to remove code that will work just to make more work for someone else later.

Reply:
Do our Windows CI runners have dev mode enabled? Then again - does it matter, given that its unlikely we'd be running OpenHCL builds on windows runners? But double then-again, we might want to do that to validate a dev-facing scenario?

I dunno, lots of questions here.

But sure - no need to block this on those nebulous future questions. If we need to reconsider this approach later, we can.